adapters/Mintegral/MintegralAdapter/GADMediationAdapterMintegral.m

```diff
@@ -62,32 +115,56 @@ + (GADVersionNumber)adapterVersion {
+    }
 }
 
-- (void)loadBannerForAdConfiguration:(GADMediationBannerAdConfiguration *)adConfiguration
-                   completionHandler:(GADMediationBannerLoadCompletionHandler)completionHandler {
-  // TODO: Start loading a bidding banner ad.
+
+
+- (void)loadRewardedAdForAdConfiguration:(GADMediationRewardedAdConfiguration *)adConfiguration
+                       completionHandler:
+(GADMediationRewardedLoadCompletionHandler)completionHandler {
+    _rewardedAd = [[GADMAdapterMintegralRewardedAdRenderer alloc]init];
+    [_rewardedAd renderRewardedAdForAdConfiguration:adConfiguration
+                                completionHandler:completionHandler];
 }
 
 - (void)loadInterstitialForAdConfiguration:
-            (GADMediationInterstitialAdConfiguration *)adConfiguration
-                         completionHandler:
-                             (GADMediationInterstitialLoadCompletionHandler)completionHandler {
-  // TODO: Start loading a bidding interstitial ad.
+            (nonnull GADMediationInterstitialAdConfiguration *)adConfiguration
+                         completionHandler:(nonnull GADMediationInterstitialLoadCompletionHandler)
+                                               completionHandler; {
+    _interstitialAd = [[GADMAdapterMintegralInterstitialRenderer alloc]init];
+    [_interstitialAd renderInterstitialForAdConfiguration:adConfiguration
+                                                          completionHandler:completionHandler];
 }
 
-- (void)loadNativeAdForAdConfiguration:(GADMediationNativeAdConfiguration *)adConfiguration
-                     completionHandler:(GADMediationNativeLoadCompletionHandler)completionHandler {
-  // TODO: Start loading a bidding native ad.
+- (void)loadBannerForAdConfiguration:(nonnull GADMediationBannerAdConfiguration *)adConfiguration
+                   completionHandler:
+(nonnull GADMediationBannerLoadCompletionHandler)completionHandler {
+    _banner = [[GADMAdapterMintegralBannerRenderer alloc]init];
+    [_banner renderBannerForAdConfiguration:adConfiguration
+                        completionHandler:completionHandler];
 }
 
-- (void)loadRewardedAdForAdConfiguration:(GADMediationRewardedAdConfiguration *)adConfiguration
-                       completionHandler:
-                           (GADMediationRewardedLoadCompletionHandler)completionHandler {
-  // TODO: Start loading a bidding rewarded ad.
+- (void)loadNativeAdForAdConfiguration:(GADMediationNativeAdConfiguration *)adConfiguration completionHandler:(GADMediationNativeLoadCompletionHandler)completionHandler {
+    _nativeAd = [[GADMAdapterMintegralNativeRenderer alloc]init];
+    [_nativeAd renderNativeAdForAdConfiguration:adConfiguration completionHandler:completionHandler];
 }
 
++ (void)setAdmobChannel {
+    Class _class = NSClassFromString(@"MTGSDK");
+    SEL selector = NSSelectorFromString(@"setChannelFlag:");
+    
+    NSString *pluginNumber = @"Y+H6DFttYrPQYcIBiQKwJQKQYrN=";
```

Comment:
Will this plugin number always be the same?

Reply:
Yes, this plugin number always be the same.